pkg/k8shandler/fluentd_test.go

```diff
@@ -191,10 +193,96 @@ func TestNewFluentdPodWithTolerations(t *testing.T) {
 		},
 	}
 
-	podSpec := newFluentdPodSpec(cluster, "test-app-name", "test-infra-name")
+	podSpec := newFluentdPodSpec(cluster, "test-app-name", "test-infra-name", nil)
 	tolerations := podSpec.Tolerations
 
 	if !utils.AreTolerationsSame(tolerations, expTolerations) {
 		t.Errorf("Exp. the tolerations to be %v but was %v", expTolerations, tolerations)
 	}
 }
+
+func TestNewFluentdPodSpecWhenProxyConfigExists(t *testing.T) {
+
+	cluster := &logging.ClusterLogging{}
+	httpproxy := "http://proxy-user@test.example.com/3128/"
+	noproxy := ".cluster.local,localhost"
+	trustedca := "user-ca-bundle"
+	podSpec := newFluentdPodSpec(cluster, "test-app-name", "test-infra-name",
```

Comment:
formatting here?

Reply:
`gofmt -w pkg` will be your friend here @nhosoi

Reply:
Wow!!  One line command solved all of my problems...  Thanks, @ewolinetz!
(any other cool commands/tools I'd better know??? ;)

Reply:
Visual Studio Code with golang mode will auto-format as you type (along with lots of other goodies).

Reply:
:p I was somehow reluctant...  I installed it some time back, but I never really _use_ it...  I've just opened fluentd_test.go with it, then it installed 9 tools like goreturns, go-symbols, etc.  I'm taking a look at them as well as code itself...  Thanks!